client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/index.jsx

```diff
@@ -45,204 +40,85 @@ export class BusinessPlanUpgradeUpsell extends PureComponent {
 
 		return (
 			<header className="business-plan-upgrade-upsell__small-header">
+				<h1 className="business-plan-upgrade-upsell__small-header-title">
+					{ translate( 'Limited time offer' ) }
+				</h1>
+			</header>
+		);
+	}
+
+	image() {
+		return <img className="business-plan-upgrade-upsell__image" src={ upsellImage } alt="" />;
+	}
+
+	title() {
+		const { translate } = this.props;
+		return (
+			<>
 				<h2 className="business-plan-upgrade-upsell__title">
-					{ translate( "Hold tight, we're getting your site ready." ) }
+					{ translate( 'Upgrade your site to the most powerful plan ever' ) }
 				</h2>
-			</header>
+			</>
 		);
 	}
 
 	body() {
 		const {
 			translate,
-			planRawPrice,
-			planDiscountedRawPrice,
-			currencyCode,
-			hasSevenDayRefundPeriod,
+			// planRawPrice,
+			// planDiscountedRawPrice,
+			// currencyCode,
+			// hasSevenDayRefundPeriod,
```

Comment:
If these are being commented-out, could we just remove them instead?

Reply:
Thanks, @sirbrillig, I will delete it. I start it as a draft PR and leave it there because I will use it for the discount pill.